search-commons/src/main/java/no/unit/nva/search2/ResourceAwsQuery.java

```diff
@@ -49,7 +49,7 @@ static Builder builder() {
 
     public String doSearch(ResourceAwsClient queryClient) {
         return
-            this.getMediaType().equals(MediaType.CSV_UTF_8)
+            this.getMediaType().is(MediaType.CSV_UTF_8)
```

Comment:
nullsafe?

Reply:
Nei, men getMediaType er alltid fyllt inn

Reply:
Jeg snur uttrykket for ordens skyld